script/tool/lib/src/version_check_command.dart

```diff
@@ -156,7 +183,11 @@ class VersionCheckCommand extends PackageLoopingCommand {
   bool get hasLongOutput => false;
 
   @override
-  Future<void> initializeRun() async {}
+  Future<void> initializeRun() async {
+    _gitVersionFinder = await retrieveVersionFinder();
+    _mergeBase = await _gitVersionFinder.getBaseSha();
+    _changedFiles = await _gitVersionFinder.getChangedFiles();
```

Comment:
nit: These could be parallelized, right?  No need to serialize them. 

```
mergeBase = _gitVersionFinder.getBaseSha().then((x) { _mergeBase = x; });
...
await Futures.wait([mergeBase, changedFiles])
```

Reply:
In theory yes, but `getChangedFiles` needs the base SHA, so it would make the code more complex without actually changing anything.